imports/plugins/core/ui/client/containers/avatar.js

```diff
@@ -1,11 +1,21 @@
 import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
 import { Meteor } from "meteor/meteor";
 import { Accounts } from "/lib/collections";
+import { Reaction } from "/client/api";
 import { ReactionAvatar } from "../components/avatar";
 
 const composer = (props, onData) => {
-  const user = Accounts.findOne(Meteor.userId());
-  const email = user.emails[0].address;
+  const targetUserId = Reaction.Router.getQueryParam("userId");
+  let user = Accounts.findOne(targetUserId);
```

Comment:
To make it easier for others to understand code that relates to accounts vs users, I'd like to see us use `account` whenever we're getting an account instead of `user`

Reply:
Ok. I agree that that would probably be a better practice.